CONTRIBUTING.md

```diff
@@ -509,3 +509,7 @@ what is implemented by a pull request.
 * **S-refactor**: This is a refactor. This label describes proposed or
   implemented changes that are related to improve code quality or set up for
   future changes, but shouldn't effect behavior, fix bugs, or add new APIs.
+
+## Releases
+
+TBD: This section will describe the release process.
```

Comment:
Do you mean you will describe the new process after you release the new version?

Reply:
Sorry. Should have marked as draft. I pushed this to test the CI because something isn't working right with the release PR.

I plan to update this after we get a release out with release-plz.